install.ps1

```diff
@@ -1,5 +1,8 @@
 param([switch]$WhatIf = $false)
 
+# Dot source for Get-FileEncoding
+. $PSScriptRoot\Utils.ps1

```

Comment:
Thought I caught that.  We can just use Split-Path $MyInvocation.MyCommand.Path -Parent instead.